video/rooms/display-camera-preview/display-camera-preview.m

```diff
@@ -2,20 +2,23 @@
 capture video from the front or rear-facing device camera  */
 @property (nonatomic, strong) TVICameraCapturer *camera;
 
-// Video Tracks (local or remote) can be attached to any UIView 
-@property (weak, nonatomic) IBOutlet UIView *previewView;
+TVIVideoView *previewView = [[TVIVideoView alloc] initWithFrame:rect];
 
 /* LocalMedia represents our local camera and microphone (media) configuration
 that can be sent to other Participants in Rooms, or previewed locally */
 @property (nonatomic, strong) TVILocalMedia *localMedia;
```

Comment:
Remove these property declarations? It's confusing to mix `previewView` with the properties.